backbone.js

```diff
@@ -308,6 +308,7 @@
   Events.once = function(name, callback, context) {
     // Map the event into a `{event: once}` object.
     var events = eventsApi(onceMap, {}, name, callback, _.bind(this.off, this));
+    if (arguments.length === 2 && typeof name === 'string') callback = void 0;
```

Comment:
Good point, forgot that it causes a deopt if you modify a param. I'll change it to `context != null`